pkg/mimirpb/compat.go

```diff
@@ -238,10 +238,11 @@ func SampleJsoniterEncode(ptr unsafe.Pointer, stream *jsoniter.Stream) {
 		return
 	}
 
+	smp := *((*Sample)(ptr))
```

Comment:
[nit] Don't need this. You already have `sample := (*Sample)(ptr)` above.

Reply:
done @ 7e3ae47e6a6787189f206323179e2fc8a421686d